src/sentry/static/sentry/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -341,4 +386,4 @@ const IconUpdate = styled('div')`
   background-color: ${p => p.theme.yellow};
 `;
 
-export default withApi(SavedQueryButtonGroup);
+export default withProjects(withApi(SavedQueryButtonGroup));
```

Comment:
Probably the only thing I couldn't hide behind a feature flag. Is this okay?

Reply:
Can we use `app/utils/projects` instead? I want to move away from using `withProjects` as it assumes that all projects are currently loaded. For large orgs this can mean thousands of projects.

Reply:
Talked about on slack, will need to add project selection by id soonTM